bin/release-package.php

```diff
@@ -46,6 +46,13 @@
 }
 $tag_version = $argv[2];
 
+$command       = sprintf(
+	'bin/version-packages.sh --package=%1$s --no-update',
+	escapeshellarg( $package_name )
+);
+execute( $command, 'Could not update sub-package dependency versions.', true, true );
```

Comment:
This is just an example of where/how it can be used in this PHP script. I don't think tagging later in the php script will actually work unless the results of this are committed? It will need a little bit of work to incorporate fully, but I didn't want to start pushing random tags to the live repos :) 

@jeherve do you want to take a stab at this? In another PR if you prefer?